src/components/BookList.jsx

```diff
@@ -1,60 +1,35 @@
-import { useState } from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Kindly avoid disabling linters as we need them to write clean code and maintain a standard